lib/util.js

```diff
@@ -291,6 +306,63 @@ exports.getVideoDescription = function(html) {
 
 
 /**
+ * Get video Owner from html.
+ *
+ * @param {String} body
+ * @return {Object}
+ */
+var authorRegexp = /<a href="\/channel\/([\w-]+)"[^>]+>(.+?(?=<\/a>))/
+var aliasRegExp = /<a href="\/user\/([^"]+)/
+exports.getAuthor = function(body) {
+  var ownerinfo = this.between(body, '<div id="watch7-user-header" class=" spf-link ">', '<div id="watch8-action-buttons" class="watch-action-buttons clearfix">');
+  ownerinfo = new Entities().decode(ownerinfo)
+  if (ownerinfo === '') {
+    return {};
+  }
```

Comment:
You can move this if statement one line above, since decoding entities won't matter if nothing was found by `between`